test_ACL_Func_065 (__main__.test_ACL_Func_065) ... FAIL

======================================================================
FAIL: test_ACL_Func_065 (__main__.test_ACL_Func_065)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "ACL_Func_065.py", line 334, in test_ACL_Func_065
    self.befNAftSwitchOver()
  File "ACL_Func_065.py", line 303, in befNAftSwitchOver
    self.failIf(acl_cntrs != True, "Permit Acl counters are not incrementing ")
AssertionError: Permit Acl counters are not incrementing 

----------------------------------------------------------------------
Ran 1 test in 370.684s

FAILED (failures=1)
